### DISCUSSION TITLE
Anyone with success installing on Raspberry Pi 5?

### DISCUSSION THREAD

**Discussion by GoldenSaber** [iiab/iiab#3721] [General]
New Raspberry Pi 5, wouldn’t take existing image. Re-imaged MicroSD, same error. OS not compatible, post screen mentioned updating config.txt with os_check=0 to bypass the check but that did not work. Attempting with fresh OS install and curl script but I have only had luck with that one time and don’t remember my troubleshooting steps to get that to work correctly, plus it’s missing some minor nuances of the dedicated IIAB images. Speed of new Pi is impressive, can’t wait to get IIAB running on it.

Anyone have any luck or pointers?

**Comment by tim-moody** [comment]
I have installed on 2023-12-11-raspios-bookworm-arm64-lite.img on rpi 5. 

I don't use curl but write the image to sd card from linux, and add curl-like scripts to the image.
I also do a couple of things to make it boot with ssh on and set the pi password:

```
touch $BOOTFS_PART_MNT/ssh
cp userconf.txt $BOOTFS_PART_MNT

```
Which 'dedicated IIAB images' are you using?

**Reply by GoldenSaber** [reply]
Running the rPi imager “rpi-imager --repo http://iiab.io/fast.json” installing the medium build option.

**Reply by holta** [reply]
Apologies @GoldenSaber that Internet-in-a-Box image is very dated from months ago.

Let me post new images very shortly.

In the interim, please also consider running this immediately (e.g. today) on a clean/fresh install of Raspberry Pi OS Bookworm:

```
curl iiab.io/install.txt | bash
```

**Comment by ncnoah** [comment]
My install on my Pi-5 keeps hanging at:
TASK [luanti : Download default game https://content.luanti.org/packages/Luanti/minetest_game/ (2025-03-02 VERSION FOR NOW) -- if luanti_default_game == "minetest" AND (RasPiOS >= 13 or Debian >= 13 or Ubuntu >= 24.10)] ***
And bombing out.  There may be more wrong after that, but this is at least part of the problem.

**Comment by holta** [comment]
Discussion with @ncnoah (Luanti install issue on Raspberry Pi 5) moved to:

- #4338
